frontend/src/views/HomePage.tsx

```diff
@@ -1,9 +1,34 @@
 import React, { ReactElement } from 'react';
+import { useNavigate } from 'react-router-dom';
 
-const HomePage: React.FunctionComponent = (): ReactElement => (
-  <div>
-    <p>HOMEPAGE</p>
-  </div>
-);
+import { PATH_TO_GAMEPAGE, PATH_TO_LANDINGPAGE } from '../constants/paths';
+
+const HomePage: React.FunctionComponent = (): ReactElement => {
+  const navigate = useNavigate();
+  const views = [
+    { name: 'Play game', url: PATH_TO_GAMEPAGE },
+    { name: 'Scoreboard', url: PATH_TO_LANDINGPAGE },
+    { name: 'How to play', url: PATH_TO_LANDINGPAGE },
+    { name: 'Credits', url: PATH_TO_LANDINGPAGE },
+  ];
+  return (
+    <div className=" bg-DARKTHEME_BACKGROUND_COLOR flex justify-center mt-30 pt-10">
+      <div className="flex flex-col">
+        {views.map(({ url, name }) => (
```

Comment:
destructuring in array method parameter may be hard to read, let's stick with single parameter and then use dot notation to access name and url